manifests/j/JRSoftware/InnoSetup/6.2.1/JRSoftware.InnoSetup.locale.en-US.yaml

```diff
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.1.3 $debug=QUSU.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: JRSoftware.InnoSetup
+PackageVersion: 6.2.1
+PackageLocale: en-US
+Publisher: jrsoftware.org
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author: 
+PackageName: Inno Setup
+PackageUrl: https://jrsoftware.org/isinfo.php
+License: Copyright (C) 1997-2020 Jordan Russell. All rights reserved. Portions Copyright (C) 2000-2019 Martijn Laan. All rights reserved.
```

Comment:
A copyright isn't a license; Please update this with actual license terms such as `Proprietary`, `MIT`, `GPL-2.0`, or other terms as appropriate